app/views/return-requirements/check.njk

```diff
@@ -108,18 +108,164 @@
           preventDoubleClick: true
         }) }}
       </form>
-
       {% for requirement in requirements %}
         {# Set an easier to use index #}
-        {% set rowIndex = loop.index0 %}
+        {% set rowIndex = requirement.index %}
```

Comment:
This will defo solve our problem! The only thing I'd question is, why not just add `requirement.index` to the end of the links instead of `rowIndex`? You could then do away with `rowIndex` altogether.

Reply:
I only did it for legibility, happy with either way